helm_deploy/values-preprod.yaml

```diff
@@ -12,3 +12,4 @@ generic-service:
     API_BASE_URL_OAUTH: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth
     DATA_COMPLIANCE_DELETION_ENABLED: false
     DATA_COMPLIANCE_DECEASED_DELETION_ENABLED: false
+    DATA_COMPLIANCE_OFFENDER_NO_BOOKING_DELETION_ENABLED: false
```

Comment:
minor point - you're normally better putting values that are the same across all envs in `values.yaml` instead and then overriding in dev etc. as necessary

Reply:
Unfortunately in this case we can't test any deletions in dev but, in the future will be running them in preprod & prod